template/python-flake/flake.nix

```diff
@@ -0,0 +1,51 @@
+{
+  description = "Your jupyterWith project";
+
+  nixConfig.extra-substituters = [
+    "https://tweag-jupyter.cachix.org"
+  ];
+  nixConfig.extra-trusted-public-keys = [
+    "tweag-jupyter.cachix.org-1:UtNH4Zs6hVUFpFBTLaA4ejYavPo5EFFqgd7G7FxGW9g="
+  ];
+
+  inputs.flake-compat.url = "github:edolstra/flake-compat";
```

Comment:
What's `flake-compat` used for? Should these templates also include a `default.nix` to take advantage of it?